src/screenControl.py

```diff
@@ -375,6 +380,8 @@ def processInput(self, key):
             # before exiting the program
             self.getFilesToUse()
             sys.exit(0)
+        elif self.mode == X_MODE and key in lbls:
```

Comment:
I think that would be worse.  When X_MODE is on we want to exclude G and A from the keybindings set because they conflict with line letters.  But as of yet we only have an if else block, not a set to subtract from.  There's no case to handle separately.